apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx

```diff
@@ -58,13 +60,14 @@ const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onN
 			</div>
 			<div className="nux-launch-step__body">
 				<PlansGrid
+					currentPlanProductId={ planProductId }
 					currentDomain={ domain }
 					onPlanSelect={ handleSelect }
 					onPickDomainClick={ handlePickDomain }
 					disabledPlans={
 						hasPaidDomain
 							? {
-									[ Plans.PLAN_FREE ]: __( 'Unavailable with domain', 'full-site-editing' ),
+									[ 'Free' ]: __( 'Unavailable with domain', 'full-site-editing' ),
```

Comment:
Same as in Focused Launch https://github.com/Automattic/wp-calypso/blob/trunk/packages/launch/src/focused-launch/plan-details/index.tsx#L76
Should we consider using a constant? 🤔

Reply:
We can also use `Free` directly (instead of `[ 'Free' ]`), TypeScript is clever enough to autocomplete it and accept it as valid

Reply:
I don't think we need a constant. `disabledPlans` only accept the right keys. Thanks TS!